resources/views/movies/create.blade.php

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Try to extend the existing layout here :D. This might give an error now, if you visit as a guest, about the $name as the App layout is using a component named navigation that gets the name from the authenticated user. 

However, I do think that for this route the user should be authenticated to be able to access this page don't you think? For this to work, you can use middleware. Laravel has a middleware called auth, which will only allow the user to visit a page if the user has been authenticated. Otherwise, it will redirect to the login page. 😄

Reply:
I have set the create and update pages to only have guest.layout as middle ware ensures that they require a log in still and neither of them have access to the navigation bar (there is no need for them to have access to it)

However, I might need to consider adding a 'Cancel' or 'Return' button if the admin changes their mind about the action